### PR TITLE
Fix #129. Fix hero section

### LINKED ISSUE #129: Hero section on home different
Old:
Selection_271
Current:
Selection_270
Tasks:
- Move the "Hardfork" banner down
- Move the "mehr" with arrow to the bottom
- Make the "mehr" arrow not fat

### PR DESCRIPTION
**Pull request by kiliw**
Fix #129.
Fix hero section

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,6 +48,7 @@
     "react-dom": "^16.11.0",
     "react-helmet": "^5.2.0",
     "react-hook-inview": "^4.1.2",
+    "react-icons": "^3.8.0",
```
Comment (kiliw) [inline package.json]: I added react icons because font awesome does not provide a thin version for the downward facing arrow in the free version.

**Comment by Levino** [comment]
Got a weird jump of the hardfork banner

File: src/components/Hero/index.tsx [hunk 2]
```diff
@@ -13,32 +13,41 @@ export const HeroSection: FunctionComponent<IntroProps> = ({
...
+  const [showHeadline, setShowHeadline] = useState(false)
+  const [showSubheadline, setShowSubheadline] = useState(false)
+  const [showBackground, setShowBackground] = useState(false)
+  const [showMoreSection, setShowMoreSection] = useState(false)
 
   useEffect(() => {
-    setAnimation(true)
     setTimeout(() => {
-      setThirdAnimation(true)
+      setShowBackground(true)
```
Comment (kiliw) [inline src/components/Hero/index.tsx]: I tried using the `setShowBackground(true)` outside the `setTimeout()` but that does not work and the animations are not triggered. The component is rather rendered  in the final state and no transitions occure.

File: src/components/Hero/Hero.module.scss [hunk 3]
```diff
@@ -13,4 +17,56 @@
...
+  &::after {
+    background: #fff;
+    bottom: 0;
+    content: "";
+    display: block;
+    height: 2px;
+    position: absolute;
+    right: 0;
+    width: 0%;
+    transform: scale(0);
```
Comment (Levino) [inline src/components/Hero/Hero.module.scss]: Do we need this?

File: src/scss/global.module.scss [hunk 4]
```diff
@@ -6,4 +6,12 @@
     url("../img/banner.jpg");
   background-position: center center;
   background-size: cover;
+  &:global(.hiddenBackground) {
+      opacity: 0;
+      transition: opacity 3.5s;
+
+  }
+  &:global(.showBackground) {
+opacity: 1;
```
Comment (Levino) [inline src/scss/global.module.scss]: This does not look linted.

File: .eslintrc.json [hunk 5]
```diff
@@ -1,7 +1,8 @@
 {
   "extends": [
     "react-app",
-    "prettier"
+    "prettier",
+    "stylelint"
```
Comment (Levino) [inline .eslintrc.json]: This wont be run on scss files as long as you do not change the "lint" script in the package.json

File: src/components/Hero/Hero.module.scss (hunk 3 above)
Comment (kiliw) [reply src/components/Hero/Hero.module.scss]: Nope the scale is not necessary. I removed it.

File: src/scss/global.module.scss (hunk 4 above)
Comment (kiliw) [reply src/scss/global.module.scss]: Moved it to a separate component.

File: .eslintrc.json (hunk 5 above)
Comment (kiliw) [reply .eslintrc.json]: I removed it from the current PR. I could not get it working right away so I created #168. That way we will have a single PR for the formatting of all the styles.

**Pull request merged**